### PR TITLE
Setup and mobile first

### PR DESCRIPTION
**Pull request by jorgegoco**
-set up linters
-complete Readme file
-add images
-add html and css files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="assets/stylesheets/main.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+  <p class="welcome">Welcome</p>
+  <nav class="mobile-menu"><i class="fa fa-bars"></i></nav>
```
Comment (tugsem) [inline index.html]: - [ ] Kindly use header tag mainly for site's title or logo (or both) and the main site navigation. Implement this after:
- The logo or title should be inserted into 'a' tag. This is for three reasons: convention, semantics, and user experience(UX).

File: assets/stylesheets/main.css [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+  color: #28352f;
+}
+
+.fa-bars {
+  position: absolute;
+  top: 57px;
+  right: 9.34%;
+}
+
+.header-mobile {
```
Comment (tugsem) [inline assets/stylesheets/main.css:137]: - [ ] Kindly use flexbox to layout the elements in the header section: logo(left) and the hamburger button(right). This gives us a lot of options to align all our elements, plus, it's responsive.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    </p>
+    <div class="social-media-icons icons-top">
+      <a href="#"><img src="/assets/images/twitter.svg"></a>
+      <a href="#"><img src="/assets/images/linkedin.svg"></a>
+      <a href="#"><img src="/assets/images/medium.svg"></a>
+      <a href="#"><img src="/assets/images/github.svg"></a>
+      <a href="#"><img src="/assets/images/angellist.svg"></a>
+    </div>
+    <a href="#"><img class="chevron" src="/assets/images/chevron.png"></a>
+  </header>
```
Comment (tugsem) [inline index.html]: - [ ] Please wrap these into `section` tag as it represents a generic standalone section of the document, so use it. To layout elements in the section, again use flexbox.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+      work and
+      experience!<br> If you like what you see and have<br> a project you need coded, <br>don’t hesitate to contact me.
+    </p>
+    <div class="social-media-icons icons-top">
+      <a href="#"><img src="/assets/images/twitter.svg"></a>
+      <a href="#"><img src="/assets/images/linkedin.svg"></a>
+      <a href="#"><img src="/assets/images/medium.svg"></a>
+      <a href="#"><img src="/assets/images/github.svg"></a>
+      <a href="#"><img src="/assets/images/angellist.svg"></a>
+    </div>
```
Comment (tugsem) [inline index.html]: - [ ] The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has `a` link (a tag).

Apart from being semantically correct, the a tag will let you connect your social networks with the icons, using href. Please use them.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    <p class="header-paragraph"> I can help you build a product ,<br> feature or website Look through<br> some of my
+      work and
+      experience!<br> If you like what you see and have<br> a project you need coded, <br>don’t hesitate to contact me.
+    </p>
+    <div class="social-media-icons icons-top">
+      <a href="#"><img src="/assets/images/twitter.svg"></a>
+      <a href="#"><img src="/assets/images/linkedin.svg"></a>
+      <a href="#"><img src="/assets/images/medium.svg"></a>
+      <a href="#"><img src="/assets/images/github.svg"></a>
+      <a href="#"><img src="/assets/images/angellist.svg"></a>
```
Comment (tugsem) [inline index.html]: - [ ] In order to display your assets in other browsers you need to use relative paths in your src attributes. Kindly add these:
```suggestion
      <a href="#"><img src="./assets/images/twitter.svg"></a>
      <a href="#"><img src="./assets/images/linkedin.svg"></a>
      <a href="#"><img src="./assets/images/medium.svg"></a>
      <a href="#"><img src="./assets/images/github.svg"></a>
      <a href="#"><img src="./assets/images/angellist.svg"></a>
```

File: assets/stylesheets/main.css [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+  margin-top: 16px;
+  height: 648px;
+  background-image: url(/assets/images/header_mobile_background.png);
```
Comment (tugsem) [inline assets/stylesheets/main.css]: - [ ] Please use relative path before this background image url. It can't be seen in other browsers right now.

**Review by tugsem — CHANGES_REQUESTED** [review]
Hi @jorgegoco , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!
Alt Text
## Highlights ⭐ 
:white_check_mark: You used the right Github Flow.
:white_check_mark: Your PR messages are descriptive overall.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @jorgegoco , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**